notesapplication/notes/api_v1/serializers.py

```diff
@@ -40,36 +76,56 @@ def update(self, instance, validated_data):
         )
         return super().update(instance, validated_data)
 
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        if len(representation["comment"]) > NUMBER_TO_GET_LATEST_COMMENT:
+            last_comment = representation["comment"][
+                len(representation["comment"]) - NUMBER_TO_GET_LATEST_COMMENT
+            ]
+            representation["comment"] = []
+            representation["comment"].append(last_comment)
+        return representation
 
-class NoteVersionSerializer(serializers.ModelSerializer):
+
+class NoteCommentSerializer(serializers.ModelSerializer):
     """
-    NotesVersionSerializer : Serializer for Note version and all of operations
-                             of versioning
+    NoteCommentSerializer : Serializer for showing comments with
+                            the particular note (when retrieve method is called)
     """
 
+    comment = serializers.SerializerMethodField("paginated_comment")
+
     class Meta:
-        model = NoteVersion
-        fields = "__all__"
+        model = Note
+        fields = [
+            "id",
+            "title",
+            "text",
+            "date_created",
+            "date_updated",
+            "user",
+            "archive",
+            "sharedwith",
+            "comment",
+        ]
 
+    def paginated_comment(self, obj):
```

Comment:
Kindly move this to the pagination file. Serializers shouldn't be responsible for paginating the response.